src/Riok.Mapperly/Descriptors/MappingBuilders/EnumMappingBuilder.cs

```diff
@@ -78,11 +97,31 @@ private static TypeMapping BuildNameMapping(MappingBuilderContext ctx, bool igno
             .Where(x => x.Target != null)
             .ToDictionary(x => x.Source.Name, x => x.Target!.Name);
 
-        if (enumMemberMappings.Count == 0)
+        if (!enumMemberMappings.Any())
```

Comment:
Is this change needed? Isn't the count check faster?

Reply:
Is indeed faster than Any(), it's just my personal preference. I've no issue with changing it back :)